src/directory.rs

```diff
@@ -54,4 +54,4 @@ impl Directory
 		}
 		Ok(())
 	}
-}
+}
```

Comment:
It would likely be more appropriate to include this as a separate commit focused on cosmetic changes.